src/maidsafe/vault/version_handler/service.cc

```diff
@@ -453,29 +507,23 @@ void VersionHandlerService::HandleChurnEvent(
 
 void VersionHandlerService::TransferAccount(const NodeId& dest,
     const std::vector<Db<VersionHandler::Key, VersionHandler::Value>::KvPair>& accounts) {
-  // If account just received, shall not pass it out as may under a startup procedure
-  // i.e. existing DM will be seen as new_node in close_nodes_change
-  if (account_transfer_.CheckHandled(routing::GroupId(routing_.kNodeId()))) {
-    LOG(kWarning) << "VersionHandler account just received";
-    return;
-  }
-  std::vector<std::string> actions;
-  for (auto& account : accounts) {
-    VLOG(nfs::Persona::kVersionHandler, VisualiserAction::kAccountTransfer, account.first.name,
-         Identity{ dest.string() });
-    protobuf::DataManagerKeyValuePair kv_msg;
-    kv_msg.set_key(account.first.Serialise());
-    kv_msg.set_value(account.second.Serialise());
-    actions.push_back(kv_msg.SerializeAsString());
-    LOG(kVerbose) << "VersionHandler sent account " << DebugId(account.first.name)
-                  << " to " << HexSubstr(dest.string())
-                  << " with vaule " << account.second.Print();
+  assert(!accounts.empty());
+  protobuf::AccountTransfer account_transfer_proto;
+  {
+    for (auto& account : accounts) {
+      VLOG(nfs::Persona::kVersionHandler, VisualiserAction::kAccountTransfer,
+           account.first.name, Identity{ dest.string() });
+      protobuf::VersionHandlerKeyValuePair kv_pair;
+      kv_pair.set_key(account.first.Serialise());
+      kv_pair.set_value(account.second.Serialise());
+      account_transfer_proto.add_serialised_accounts(kv_pair.SerializeAsString());
+      LOG(kVerbose) << "account.first send account " << HexSubstr(account.first.name.string())
```

Comment:
I think the decision was to remove Info and Verbose log entries from the code and only Warning Error should be left and only when they are really Warnings or above. Can you remove these Info and Verbose please Qi.